docs/source/1.0/spec/aws/aws-json-1_0-protocol.rst

```diff
@@ -57,8 +87,53 @@ See
             }
         }
 
+The following example defines a service that requires the use of
+``h2`` when using event streams.
+
+.. code-block:: smithy
+
+    namespace smithy.example
+
+    use aws.protocols#awsJson1_0
+
+    @awsJson1_0(eventStreamHttp: ["h2"])
```

Comment:
Is this valid? The docs imply that `http` needs to be set if eventStreamHttp is set.